eng/format-job.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  containerName: ''
+  crossrootfsDir: ''
+  timeoutInMinutes: ''
+
+### Format job
+jobs:
+- template: xplat-job.yml
+  parameters:
+    buildConfig: ${{ parameters.buildConfig }}
+    archType: ${{ parameters.archType }}
+    osGroup: ${{ parameters.osGroup }}
+    osIdentifier: ${{ parameters.osIdentifier }}
+    containerName: ${{ parameters.containerName }}
+    crossrootfsDir: ${{ parameters.crossrootfsDir }}
+    timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+    name: ${{ format('format_{0}_{1}', parameters.osIdentifier, parameters.archType) }}
+    displayName: ${{ format('{0} {1} Formatting', parameters.osIdentifier, parameters.archType) }}
```

Comment:
Can you please make displayName to be `Formatting OS Arch` to keep the job naming consistency